include/eve/detail/top_bits.hpp

```diff
@@ -50,7 +72,8 @@ EVE_FORCEINLINE auto movemask_raw(Logical p)
   using ABI = abi_type_t<Logical>;
   using e_t = element_type_t<Logical>;
 
-  if constexpr( std::same_as<ABI, x86_128_>)
+       if constexpr( eve::current_api == avx512 ) return p.storage();
```

Comment:
This works but we usually check for `!ABI::is_wide_logical` just in case Intel drop us something strange later on